tests/ci/sign_release.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+import sys
+import os
+import logging
+from env_helper import GPG_BINARY_SIGNING_KEY, GPG_BINARY_SIGNING_PASSPHRASE, TEMP_PATH, REPO_COPY, REPORTS_PATH
+from s3_helper import S3Helper
+from pr_info import PRInfo
+from build_download_helper import download_builds_filter
+import hashlib
+
+
+CHECK_NAME = "Sign release (actions)"
+
+def hash_file(file_path):
+    BLOCK_SIZE = 65536 # The size of each read from the file
+
+    file_hash = hashlib.sha256() # Create the hash object, can use something other than `.sha256()` if you wish
+    with open(file_path, 'rb') as f: # Open the file to read it's bytes
+        fb = f.read(BLOCK_SIZE) # Read from the file. Take in the amount declared above
+        while len(fb) > 0: # While there is still data being read from the file
+            file_hash.update(fb) # Update the hash
+            fb = f.read(BLOCK_SIZE) # Read the next block from the file
+
+    hash_file_path = file_path + '.sha512'
```

Comment:
Why is it `.sha512` in file name, while we use `sha256` ?

Reply:
Because it used to be `sha512` and I forgot to update the filename lol, will update it